client/src/components/AlertBanner/AlertBanner.tsx

```diff
@@ -8,7 +8,7 @@ interface AlertBannerProps {
   children: React.ReactNode;
   banner_class: string;
   additional_class_names: string;
-  style: Object;
+  style: Record<string, unknown>;
```

Comment:
`React.CSSProperties` is the right type for React `style` objects (it's great, gives real detailed type checking for all styles).